index.html

```diff
@@ -0,0 +1,124 @@
+            disinfectants, and other applied science.
+          </li>
+          <li>
+            <strong>1942</strong> - Receives a Ph.D. in Genetics and Plant
+            Pathology
+          </li>
+          <li>
+            <strong>1944</strong> - Rejects a 100% salary increase from Dupont,
+            leaves behind his pregnant wife, and flies to Mexico to head a new
+            plant pathology program. Over the next 16 years, his team breeds
+            6,000 different strains of disease resistent wheat - including
+            different varieties for each major climate on Earth.
+          </li>
+          <li>
+            <strong>1945</strong> - Discovers a way to grown wheat twice each
+            season, doubling wheat yields
+          </li>
+          <li>
+            <strong>1953</strong> - crosses a short, sturdy dwarf breed of wheat
+            with a high-yeidling American breed, creating a strain that responds
+            well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+          </li>
+          <li>
+            <strong>1962</strong> - Visits Delhi and brings his high-yielding
+            strains of wheat to the Indian subcontinent in time to help mitigate
+            mass starvation due to a rapidly expanding population
+          </li>
+          <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
+          <li>
+            <strong>1983</strong> - helps seven African countries dramatically
+            increase their maize and sorghum yields
+          </li>
+          <li>
+            <strong>1984</strong> - becomes a distinguished professor at Texas
+            A&M University
+          </li>
+          <li>
+            <strong>2005</strong> - states "we will have to double the world
+            food supply by 2050." Argues that genetically modified crops are the
+            only way we can meet the demand, as we run out of arable land. Says
+            that GM crops are not inherently dangerous because "we've been
+            genetically modifying plants and animals for a long time. Long
+            before we called it science, people were selecting the best breeds."
+          </li>
+          <li><strong>2009</strong> - dies at the age of 95.</li>
+        </ul>
+        <p>
+          "Borlaug's life and achievement are testimony to the far-reaching
+          contribution that one man's towering intellect, persistence and
+          scientific vision can make to human peace and progress."
+          <br />
+          <br />
+
+          -- Indian Prime Minister Manmohan Singh
+        </p>
+      </section>
+
+      <h3>
+        If you have time, you should read more about this incredible human being
+        on his
+        <a
+          target="_blank"
+          id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+          >Wikipedia entry.</a
+        >
+      </h3>
```

Comment:
Kindly consider wrapping these tags with a footer tag.

Reply:
As I previously comment, I would suggest again the following for this comment too:

✨ Maintain a polite and constructive tone: Instead of using imperative statements, try phrasing your comments as suggestions to maintain a respectful and collaborative atmosphere.

🔍 Provide explanations for your requests: It's important to include the 'why' behind your suggested changes. This helps the developer understand the reasoning and importance behind the requested modification.

💡 Ensure clarity in your comments: Make sure your comments are clear and easy to understand. Provide specific details and examples to help the developer grasp your suggestions more effectively.

By following these guidelines, you'll be able to provide feedback that is both helpful and aligned with the established communication rules. Keep up the great work! 💪

Reply:
- [ ] Please provide the reason behind your suggestion because it would enhance understanding 🧠 and facilitate informed decision-making 🌳

Reply:
> - [ ] Hello @JhonatanErosa, I wanted to bring to your attention that the changes requested by the previous reviewer have not been implemented yet. I kindly request you address those changes as soon as possible.👍When you explain the reasons behind a requested change, it helps the student understand the specific problem or improvement that needs to be addressed. By providing context and reasoning, you ensure that the student comprehends the purpose of the request.✔